src/MicroOrm.Dapper.Repositories/SqlGenerator/ExpressionHelper.cs

```diff
@@ -271,15 +270,15 @@ public static string GetPropertyNamePath(Expression expr, out bool nested)
             var path = new StringBuilder();
             var memberExpression = GetMemberExpression(expr);
             var count = 0;
-            do
+            while (memberExpression != null)
```

Comment:
Why you need update this?

Reply:
`GetMemberExpression` return nullable.

Reply:
I don't understand the logic here, is it the first time `memberExpression` must not be null?